sys/vane/jael.hoon

```diff
@@ -1142,11 +1241,25 @@
       |=  [who=ship dif=diff-hull]
       ^+  [kyz ..file]
       =-  ::TODO  =; with just the type
-        :-  ?:  ?=(%& -.new)
-              (file-keys who p.new)
-            ?:  p.new  kyz
-            (file-discontinuity who)
-        ..file(hul.eth (~(put by hul.eth) who hel))
+        ?:  ?=(%& -.new)
+          :-  (file-keys who p.new)
+          ..file(hul.eth (~(put by hul.eth) who hel))
+        ?:  p.new  
+          :-  kyz
+          ..file(hul.eth (~(put by hul.eth) who hel))
+        :-  (file-discontinuity who)
+        %=  ..file
+          hul.eth  (~(put by hul.eth) who hel)
```

Comment:
Probably just `=.  hul.eth` right after the `=-`.

Reply:
We change it in like every branch here. (^:

Reply:
Refactored